hadoop-tools/hadoop-dynamometer/hadoop-dynamometer-workload/src/main/java/org/apache/hadoop/tools/dynamometer/workloadgenerator/audit/AuditLogDirectParser.java

```diff
@@ -125,8 +125,9 @@ public AuditReplayCommand parse(Text inputLine,
       relativeTimestamp = dateFormat.parse(m.group("timestamp")).getTime()
           - startTimestamp;
     } catch (ParseException p) {
-      throw new IOException("Exception while parsing timestamp from audit log",
-          p);
+      throw new IOException(
+          "Exception while parsing timestamp from audit log line: `"
```

Comment:
You have an extra backtick here. There should either be no backticks or one before and one after the `inputLine`.